### ISSUE TITLE
feat(SelfAvatar): add monochrome variant

### ISSUE THREAD

**Issue by dariadia** [dariadia/danni-s-folio#29]
### What
Users with the black-and-white theme see the `<SelfAvatar />` in black and white.

### Why
UX. WCAG.

### How
- Add theme switch variant;
- Switch between kinds based on user theme.

### Context
- SelfAvatar
- theme
